assemble.py

```diff
@@ -134,15 +133,14 @@ def copy_container_images_to_wic(target: FactoryClient.Target, factory: str, ost
 
 
 def copy_container_images_from_archive_to_wic(target: FactoryClient.Target, app_image_dir: str, app_preload_dir: str,
-                                              wic_image: str, token: str, apps_shortlist: list, progress: Progress):
+                                              wic_image: str, token: str, progress: Progress):
 
     p = Progress(2, progress)
     target_app_store = ArchiveTargetAppsStore(app_image_dir)
-    target.shortlist = apps_shortlist
     if not target_app_store.exist(target):
         logger.info('Container images have not been found, trying to obtain them...')
         apps_fetcher = TargetAppsFetcher(token, app_preload_dir)
-        apps_fetcher.fetch_target_apps(target, apps_shortlist)
+        apps_fetcher.fetch_target_apps(target, target.shortlist)
```

Comment:
That's great. I think the change won't impact the LmP's preload case since the script uses the Target object attribute to pass a shortlist, too. https://github.com/foundriesio/ci-scripts/blob/master/lmp/preload-app-images#L69.